ui/src/main/java/edu/wpi/grip/ui/pipeline/StepController.java

```diff
@@ -82,9 +103,21 @@ private void initialize() {
         new Image(InputStream.class.cast(icon))));
     buttons.getChildren().add(0, exceptionWitnessResponderButtonFactory.create(step, "Step Error"));
 
+    if (step.getInputSockets().stream()
+        .allMatch(inputSocket -> inputSocket.getSocketHint().getView()
+            .equals(SocketHint.View.NONE))) {
```

Comment:
This is saying if there is no user interaction in this step do not show the arrow